sql/hive/src/test/resources/sqlgen/broadcast_hint_nested_query_1.sql

```diff
@@ -0,0 +1,7 @@
+-- This file is automatically generated by LogicalPlanToSQLSuite.
+SELECT /*+ MAPJOIN(parquet_t0) */ *
+FROM (SELECT id tid FROM parquet_t0) T
+JOIN (SELECT key uid FROM parquet_t1) U
+ON tid=uid
+--------------------------------------------------------------------------------
+SELECT `gen_attr` AS `tid`, `gen_attr` AS `uid` FROM (SELECT `gen_attr`, `gen_attr` FROM (SELECT /*+ MAPJOIN(parquet_t0) */ `gen_attr` AS `gen_attr` FROM (SELECT `id` AS `gen_attr` FROM `default`.`parquet_t0`) AS gen_subquery_0) AS T INNER JOIN (SELECT `gen_attr` AS `gen_attr` FROM (SELECT `key` AS `gen_attr`, `value` AS `gen_attr` FROM `default`.`parquet_t1`) AS gen_subquery_1) AS U ON (`gen_attr` = `gen_attr`)) AS gen_subquery_2
```

Comment:
Yep. I'm trying to recover them in https://github.com/apache/spark/pull/14257 , too.